src/dialog-user/services/dialogData.spec.ts

```diff
@@ -124,5 +122,16 @@ describe('DataTableSettingsService test', () => {
     const testSorted = dialogData.updateFieldSortOrder(testDropDown);
     const expectedResult = [[5, 'Test2'], [2, 'Test5'], [0, 'Test']];
     expect(testSorted).toEqual(expectedResult);
+    const testDropDownDescription = {
+      values: [
+        [0, 'B'],
+        [5, 'C'],
+        [2, 'A']
+      ],
+      options: { sort_by: 'description', sort_order: 'descending' }
+    };
+    const testSortedDescription = dialogData.updateFieldSortOrder(testDropDownDescription);
+    const expectedSortedResult = [[5, 'C'], [0, 'B'], [2, 'A']];
+    expect(testSortedDescription).toEqual(expectedSortedResult);
```

Comment:
I think I'd rather see this as a separate `it` block, but I don't care that much it's not a necessary change, just wanted to point it out.

Reply:
Yeah, sometimes I like to group tests together.  I think it should be fine.